sage-ios/SAGE/SAGE/CheckinRequestTableViewCell.swift

```diff
@@ -114,13 +117,13 @@ class CheckinRequestTableViewCell: UITableViewCell {
         
         self.checkButton.setHeight(42)
```

Comment:
maybe set this as a constant in the top of the file? seems like you use 42 a lot. i know this is also done for the signuprequest cells